packages/material-ui/src/Paper/Paper.test.js

```diff
@@ -73,6 +72,9 @@ describe('<Paper />', () => {
 
   it('allows custom elevations via theme.shadows', () => {
     const theme = createMuiTheme();
+    // Theme.shadows holds a reference to `@material-ui/core/styles#shadows`
+    // Mutating it causes side effects in other tests
+    theme.shadows = theme.shadows.slice();
```

Comment:
Ooh, this looks like a bug!

Reply:
Linking it to #22517

Reply:
Already discussed in https://github.com/mui-org/material-ui/pull/22494#discussion_r483958633